src/UI/Components/Diagram/helpers.test.ts

```diff
@@ -95,21 +95,21 @@ describe("createConnectionRules", () => {
           lowerLimit: 1,
           upperLimit: 1,
           modifier: "r",
-          type: "embedded",
+          kind: "embedded",
```

Comment:
A specific reason why you went for a lower-cased value here? In the rest of the code base the kind field contains a title-cased value.